content/cybersecurity/concepts/cyber-attack/terms/ddos-attack/ddos-attack.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: 'DDoS Attack'
+Description: 'A Distributed Denial of Service(DDoS) attack occurs when a person uses sources from multiple online locations to harm a business's online operations.'
+Subjects:
+  - 'Cybersecurity'
+  - 'Computer Science'
+Tags:
+  - 'Cybersecurity'
+  - 'Computer Science'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/cyber-resilience-risk-management'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+
+A **DDoS** is a [cyber attack](https://www.codecademy.com/resources/docs/cybersecurity/cyber-attack) where a company or organization's websites and servers are unable to function effectively due to their network services are disrupted in an attempt to access their resources until they are no longer needed, ultimately leading to said websites and servers to stop working altogether.
+
+## Types of DDOS Attacks
+
+DDoS attacks typically come in three categories:
+
+1. Volumetric(raw attack volume) attacks: Utilizes methods to generate volumes of online traffic to a company's website, causing its' bandwidth to be saturated and the flow of online viewership to be disjointed.
+2. Protocol attacks: Destroys the processing capacity of network infrastructure resources like servers, firewalls, and load balancers. This happens when malicious connection requests target by Layer 3 and 4 protocol connection requests.
+3. Application Attacks: Opens connections and initiating process and transaction requests that consume finite resources like disk space and memory storage. This occurs when weaknesses in the application layer are manipulated
+
+## Signs of a DDOS-infected device
```

Comment:
It might be helpful to have a sentence here, or in the first section, describing more explicitly what a botnet is and how they are used in DDOS attacks.